dir.props

```diff
@@ -80,6 +72,15 @@
     <Platform Condition="'$(Platform)' == 'amd64'">x64</Platform>
   </PropertyGroup>
 
+  <!--
+     Switching to the .NET Core version of the BuildTools tasks seems to break numerous scenarios, such as VS intellisense and resource designer
+     as well as runnning the build on mono. Until we can get these sorted out we will continue using the .NET 4.5 version of the tasks.
+     -->
+  <PropertyGroup>
+    <BuildToolsTargets45 Condition="'$(BuildOS)' != 'Windows_NT'">false</BuildToolsTargets45>
```

Comment:
Is this affected by the change of BuildOS default to "AnyOS"?  Do we fully understand the implication to Windows builds?  I'd prefer not to affect those until this change is more baked?